Tests/TestingApps/PaywallsTester/PaywallsTester/Views/UpsellView.swift

```diff
@@ -24,11 +24,9 @@ struct UpsellView: View {
             requiredEntitlementIdentifier: Configuration.entitlement,
             purchaseStarted: { _ in
                 print("Purchase started")
-            },
-            purchaseCompleted: { _ in
+            }, purchaseCompleted: { customerInfo in
```

Comment:
`customerInfo` can still be ignored here?

```suggestion
            }, purchaseCompleted: { _ in
```